pages/home.jsx

```diff
@@ -35,27 +35,36 @@ var FeaturedPost = React.createClass({
   propTypes: {
     data: React.PropTypes.object.isRequired
   },
+  componentWillUpdate: function(nextProps, nextState) {
+    this.parsedMomentDate = moment(new Date(nextProps.data.publishedDate));
```

Comment:
> why not just check during `render` to see if `props.data.publishedDate` is truthy or non-empty?

This was my very initial attempt, and that didn't work for some reason I couldn't figure out.  Guess I've probably confused myself with the nested `data` object. Heh, anyways, now that it's working I'll update this PR again!